android/README.md

```diff
@@ -86,9 +86,9 @@ or to enable the QTI backend:
 ```
 make WITH_QTI=1 app
 ```
-or to enable both the MediaTek and QTI backends:
+or to enable multiple backends:
 ```
-make WITH_QTI=1 WITH_MEDIATEK=1 app
+make WITH_QTI=1 WITH_SAMSUNG=1 WITH_PIXEL=1 app
```

Comment:
No-one but MLCommons can build the Samsung backend. Is it clear you would need to get the Samsung SDK (and Qualcomm SDK) to run this command? Why not WITH_MEDIATEK=1? Neuron delegate is not (and should not be) built by default.

Reply:
I just wanted to show an example command to build multiple backends as stated in the text above, we can also add another text stated that building other backends require further setup described by each vendor.
Or we can leave the example command unchanged.

Reply:
BTW, if Samsung backend is not public, should we remove the folder `mobile_back_samsung` to avoid confusion for public users?

Reply:
The glue code is public but SDK and Makefile are only available under NDA

Reply:
Only glue code (excluding Makefile) is public